server/aws-lsp-codewhisperer/src/language-server/localProjectContext/initializeUtils.ts

```diff
@@ -0,0 +1,29 @@
+import { InitializeParams, WorkspaceFolder } from '@aws/language-server-runtimes/protocol'
+import { Logging } from '@aws/language-server-runtimes/server-interface'
+import * as path from 'path'
+import { URI } from 'vscode-uri'
+
+export function getWorkspaceFolders(logging: Logging, params?: InitializeParams): WorkspaceFolder[] {
```

Comment:
why is this in localProjectContext? instead of shared/ ? seems generally applicable , not specific to project context.